frappe/desk/doctype/event/event.py

```diff
@@ -55,6 +55,13 @@ def validate(self):
 		if self.sync_with_google_calendar and not self.google_calendar:
 			frappe.throw(_("Select Google Calendar to which event should be synced."))
 
+		if not self.sync_with_google_calendar or self.event_category not in [
+			"Event",
+			"Meeting",
+			"Other",
+		]:
+			self.add_video_conferencing = False
```

Comment:
let `add_video_conferencing` show on all event categories? - we show the `sync_with_google_calendar` checkbox by default anyway (?)

```suggestion
		if not self.sync_with_google_calendar:
			self.add_video_conferencing = 0
```